integration_tests/test_search.py

```diff
@@ -1854,6 +2190,76 @@ def test_witches(self):
             Common(subject=re.compile('witch'))
         )
 
+class TestFuzzyConfounders(SearchTest):
+    """Test searches on very distinct terms that are near each other in
+    Levenstein distance.
+    """
```

Comment:
This verifies that it's okay to remove the special fuzzy regexp matching in core.